src/mixins/controls.js

```diff
@@ -163,6 +204,11 @@ const OrbitControlsMixin = (superclass) => class extends superclass {
         this.scene2.x = x;
         this.scene2.y = y;
         this.scene2.z = z;
+
+        //axis labels
+        this.scene2.labelX = labelX;
+        this.scene2.labelY = labelY;
+        this.scene2.labelZ = labelZ;
```

Comment:
It is probably a good idea to add secondAxesGroup as a 3D object group the x, y, z and the corresponding labels there for brevity